src/main/java/org/jfrog/hudson/pipeline/common/types/Docker.java

```diff
@@ -39,13 +40,17 @@ public void setHost(String host) {
         this.host = host;
     }
 
+    public void setJavaArgs(String javaArgs) {
+        this.javaArgs = javaArgs;
+    }
+
     public void setServer(ArtifactoryServer server) {
         this.server = server;
     }
 
     @Whitelisted
     public Docker addProperty(String key, String... values) {
-        properties.putAll(key, Arrays.asList(values));
+        properties.put(key, String.join(",", values));
```

Comment:
It is worth testing manually that adding a property with multiple values actually works, and that the values are not saved in Artifactory as a one comma separated value.